### ISSUE TITLE
Receiving metadata mismatch error when returning received jwt for a verified code.

### ISSUE THREAD

**Issue by johnschoeman** [google/exposure-notifications-verification-server#103]
### TL;DR

After hitting the verify endpoint, `api/verify`, and receiving a verification token, we post to the get-certificate endpoint, `api/certificate`,  with the token and our Hmac of our exposure keys and we are getting the following error:

```
400: {"error":"verification token test metadata mismatch"}
```

Looking for guidance on if this is a bug in the server logic when looking up the token, or if we are misunderstanding the intended flow here.

**Expected behavior**
Given that we are receiving the token directly from the server and returning it un modified, we expect that the server to not error with a mismatch of token metadata.

**Observed behavior**
We received the error.

### Reproduction

1. created a verification code via the web portal we are hosting
2.  send the verification code to the `api/verify` endpoint with the API token in the X-API-key header
3.  receive the jwt back from the server.
4. create an HMAC with some TEKs and a random key generated by the client
5. post to `api/certificate` with a data payload: `{ token: the_returned_token, ekeyhamc: the_generated_hmac }`

**Environment**
Not sure

**Additional information**
when we decoded the token we got this:
```
{
  "aud": "diagnosis-verification-example",
  "exp": 1594408633,
  "jti": "obfuscated jti",
  "iat": 1594322233,
  "iss": "diagnosis-verification-example",
  "sub": "confirmed."
}
```

one thing we noticed was the `.` dot at the end of `confirmed`.

**Comment by mikehelmick** [comment]
Do you have any sever logs you can share?

**Comment by mikehelmick** [comment]
Also - can you confirm that you are on the latest version - I may just fixed this [yesterday](https://github.com/google/exposure-notifications-verification-server/commit/21e98b40f7c6679d3d5f51419ec1642db7c6ea81)

**Comment by johnschoeman** [comment]
@mikehelmick , thanks for the quick response! I confirmed with @sherifkozman that we are likely not on the most recent version. I will wait till that is upgraded and give it another pass on the client side.

**Comment by mikehelmick** [comment]
sounds good - let me know if the latest version works

/assign @johnschoeman

**Comment by johnschoeman** [comment]
@mikehelmick , the latest version seems to be working!  Thank you for your help, going to close the issue as it has been resolved for us.
